components/kyma-environment-broker/internal/suspension/handler.go

```diff
@@ -125,6 +125,10 @@ func (h *ContextUpdateHandler) suspend(instance *internal.Instance, log logrus.F
 }
 
 func (h *ContextUpdateHandler) unsuspend(instance *internal.Instance, log logrus.FieldLogger) error {
+	if instance.IsExpired() {
+		log.Info("Expired instance cannot be unsuspended")
```

Comment:
nit: maybe return this as an error so the caller knows not to keep trying? I can imagine being confused why my `PATCH` with `active: true` is returning OK 200 but nothing is happening and my instance is still suspended

Reply:
it is by design. Expiration is not triggered by BTP (user). From BTP point of view it would be a bug - we do not accept unsuspension.